lib/db.js

```diff
@@ -476,43 +467,43 @@ DB.prototype._createTable = function (keyspace, schema, tableName) {
     });
 
     // Finally, create the main data table
-    var cql = 'create table if not exists '
+    var sql = 'create table if not exists '
         + dbu.cassID(keyspace + '_' + tableName) + ' (';
     for (var attr in schema.attributes) {
         var type = schema.attributes[attr];
-        cql += dbu.cassID(attr) + ' ';
+        sql += dbu.cassID(attr) + ' ';
         switch (type) {
```

Comment:
I concur with @d00rman that focusing on the basics should be helpful. 

In the cassandra backend, [the type mapping is now moved to dbUtil](https://github.com/wikimedia/restbase-mod-table-cassandra/blob/master/lib/dbutils.js#L283), which makes it usable from several places. This might be a good idea for SQLite too.